### DISCUSSION TITLE
다운로드부터 안되네요 ㅠㅠ404에러..?

### DISCUSSION THREAD

**Discussion by sjklfjlksfjl** [changdoc/naver-vaccine-macro#93] [Q&A] [answered]
크롬웹스토어 설치버튼을 누르면 아래와 같은 에러가 뜹니다.ㅜ
크롬 확장프로그램에서 [NAVER Vaccine MACRO] 를 검색해봐도 안뜨는데 짤린건가요?

404. 오류가 발생했습니다.
요청하신 URL을 서버에서 찾을 수 없습니다. 다른 원인은 확인할 수 없습니다.

**Comment by changdoc** [comment, marked as answer]
신규 버전 심사가 오래 걸려 동작하지 않는 버전이 게시되어 있는 상태라 스토어에서는 내려둔 상태입니다.

https://github.com/changdoc/naver-vaccine-macro/issues/29
여기 내용 확인하시면 최신 버전 설치해서 사용 가능합니다.

**Reply by sjklfjlksfjl** [reply]
바로 답변해주셔서 정말 감사합니다:)
